lib/acts_as_taggable_on/tag.rb

```diff
@@ -21,7 +21,10 @@ def validates_name_uniqueness?
     ### SCOPES:
     scope :most_used, ->(limit = 20) { order('taggings_count desc').limit(limit) }
     scope :least_used, ->(limit = 20) { order('taggings_count asc').limit(limit) }
-
+    scope :skill_tags, -> { where(category: 'skill') }
```

Comment:
@usama-ashraf don't do that usama, use parameters, never hard code something like that. The gem doesn't know what are categories. These categories are specific to our app, not to the gem.

Do something like

`scope :all_tags, -> (category, enabled = true)`

I don't know the syntax.

Reply:
added changes @fbrunel